src/utils/Constants.ts

```diff
@@ -13,4 +13,18 @@ export class Constants {
     static readonly EMAIL_LOGGER_SVC_NAME = "SendEmailHandler";
 
     static readonly EMAIL_METRICS_NAMESPACE = "F2F-CRI";
+
+static readonly BASE_EVENT_TYPES = ["F2F_YOTI_START", "IPV_F2F_CRI_VC_CONSUMED", "AUTH_DELETE_ACCOUNT"];
```

Comment:
Have you linted this file? It seems to have some inconsistent indentation

Reply:
Yeah we always have problems with these files on PRs, npm run lint:fix does this but it looks fine on VSCode
<img width="876" alt="image" src="https://user-images.githubusercontent.com/13416125/235954395-b1cced0a-216f-4ca7-982e-000f09096f3e.png">